.travis.yml

```diff
@@ -1,8 +1,7 @@
 language: rust
 
 rust:
-  - nightly-2018-07-24  # pinned toolchain for clippy
-  - 1.26.0              # minimum supported toolchain
+  - nightly-2019-09-13  # pinned toolchain for clippy
```

Comment:
You can pin this to `1.38.0` now that clippy is available on the stable channel.